samples/tab-stage-view/nodejs/Readme.md

```diff
@@ -163,6 +171,18 @@ Also update `BaseUrl` according to your code runtime environment.
 
 ![AppOffice](Images/AppOffice.png) 
 
+**After opening Outlook web, click the "New mail" button.**
+
+![Open New Mail](Images/OpenNewMail.png)
+
+**on the tool bar on top,select Apps icon. Your sideloaded app title appears among your installed apps**
```

Comment:
**On the tool bar on top, select Apps icon. Your sideloaded app title appears among your installed apps**

Reply:
@Mohammed-MSFT - This is not fixed

Reply:
added